internal/notif/telegram/client.go

```diff
@@ -81,19 +101,25 @@ func (c *Client) Send(entry model.NotifEntry) error {
 		return err
 	}
 
-	for _, chatID := range chatIDs {
-		_, err := bot.Send(tgbotapi.MessageConfig{
-			BaseChat: tgbotapi.BaseChat{
-				ChatID: chatID,
-			},
-			Text:                  string(body),
-			ParseMode:             "markdown",
-			DisableWebPagePreview: true,
-		})
+	for i, chatID := range chatIDs {
+		var chatTopic int64
+		if len(chatTopics) > i {
+			chatTopic = chatTopics[i]
+		}
```

Comment:
I don't think we should use a second array to define topics. That's error prone imo.

I think we could either use a separator in chat ID to define topics like `123456789:25` where `123456789` is the chat ID and `25` an optional topic/thread id.

Also I think a chat ID can have multiple topics right?

If that's the case we could either define comma separated list of topic IDs in chat ID like `123456789:25,12` or keep `ChatTopics` that would be a map like:

```yaml
chatTopics:
  123456789:
    - 25
    - 12
```

I would tend to reuse chatIDs with `123456789:25,12`

Reply:
> I think we could either use a separator in chat ID to define topics like 123456789:25 where 123456789 is the chat ID and 25 an optional topic/thread id.

Unless I'm misunderstanding something, that would mean we'd have to change the chatID to a string, right? That would be a rather breaking change and I'm not sure if that is the best solution.

I made a draft of using a map for the chat topics in 1915c2729764e5f303e676769ea306f999034259. It's working but it has the downside that you have to make the map keys explicitly a string (`github.com/crazy-max/gonfig` doesn't support `int64` as map keys). 

Let me know what you think!